### ISSUE TITLE
Doxgen configs need updating

### ISSUE THREAD

**Issue by MarkCallow** [KhronosGroup/KTX-Software#341]
I updated Doxygen on my Windows box from 1.8.15 to 1.8.20 to get fixes to remove some bogus confusion about variables with the same name in different apps. Now I have 3 new warnings indicating that the CMake generated Doxyfiles, for example Doxyfile.ktxtools, have these outdated options:

* `TCL_SUBST`
* `PERL_PATH`
* `MSCGEN_PATH`

I cannot find out where these are coming from. These are not set in `cmake/docs.cmake`. 

Do we need an updated version of CMake to match Doxygen? Please figure out and fix the problem?

**Comment by atteneder** [comment]
Which CMake version do you use?

I don't get warnings on macOS/CMake 3.18.2.

**Comment by MarkCallow** [comment]
On macOS I have cmake 3.18.3 and Doxygen 1.8.17. I do not get warnings. The warnings started on my Windows box when I updated Doxygen to 1.8.20. Which version of Doxygen do you have on Mac? I have cmake version 3.18.0-rc2 on Windows. I'll try installing 3.18.3 later today. I don't want to update Doxygen on my Mac until we figure this out.

**Comment by atteneder** [comment]
Should've mentioned it right away:

```
$ doxygen --version
1.8.20
```

Will try Windows later

**Comment by MarkCallow** [comment]
The problem appears related to the cmake VS 2015 generator. I updated. I updated my Windows box to cmake 3.18.4 and still get the warnings but only in the VS2015 build. They don't appear in the VS2017 build. I'm testing by rebuilding just the ktxtools.doc.

Actually I'm having a lot of problems building with VS2015. The worst is errors when compiling anything using vulkan.hpp.  I don't understand why. I don't see these issues in the CI build. I wonder if the CI build is actually using VS2015.
